src/components/onCall/oncall-paid-status-dialog.tsx

```diff
@@ -74,10 +96,14 @@ const OnCallHandler = ({
       <DialogContent dividers>
         <Box sx={{ mb: 2 }}>
           <Typography variant="subtitle1">
-            <b>Person:</b> {onCallEntry.username}
+            <Typography component="span" fontWeight="bold" display="inline">
+              {strings.oncall.person}:
+            </Typography> {onCallEntry.username}
```

Comment:
Generally its a good idea to avoid unnecessarily nesting components, especially Typography. This can just be achieved with 2 sibling typographies and aligned using the styles